### ISSUE TITLE
Print intervals as `3..4`?

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalArithmetic.jl#319]
Maybe it would be less confusing to print out intervals as `3..4` instead of `[3, 4]`?

**Comment by dpsanders** [comment]
Or using brackets with 2 vertical bars?

cc @yurivish

**Comment by lbenet** [comment]
Why not having both options, so the used can decide? We already have `@format :midpoint`...

**Comment by Kolaru** [comment]
The `..` format may be kind of confusing with decimal numbers: `-3.21..5.67` (dots everywhere!).

**Comment by dpsanders** [comment]
Yes I agree that we should have both options available.

I actually find `-3.21..5.67` quite readable myself. Or add a space:
```
julia> -3.21 .. 5.67
[-3.21001, 5.67001]
```

**Comment by lbenet** [comment]
Yes, add spaces!

**Comment by perrutquist** [comment]
The `repr` function falls back to `show`, and `repr(x)` is often Julia code that creates `x`, so it would seem logical to print out either `3 .. 4` or `3.5 ± 0.5`. (Especially since `[3, 4]` creates a vector.)

I'd personally find it really useful if I could copy-paste from the output and get a new interval with a guarantee to contain the original interval (even if it does not recreate it exactly).

Whether or not to add spaces could be determined by `get(io, :compact, true)`. That's how it's done for complex numbers.

**Comment by dpsanders** [comment]
Note that you can already have the midpoint-radius version:
```jl
julia> @format midpoint
Display parameters:
- format: midpoint
- decorations: false
- significant figures: 6

julia> 3..4
3.5 ± 0.5
```

**Comment by dpsanders** [comment]
That is implemented in `@format full`, but agreed that `repr` should probably return that.

**Comment by Kolaru** [comment]
Fixed in #555
